apis/eventstream/index.yaml

```diff
@@ -66,7 +66,7 @@ get:
               description: The node has received an Attestation (from P2P or API) that passes validation rules of the `beacon_attestation_{subnet_id}` topic
```

Comment:
Once electra goes live, Lodestar will only emit this event for aggregate attestations either received via `beacon_aggregate_and_proof` gossip topic or the attestations part of the block when importing.

Curious how other client use the event but the description might be misleading since for unaggregated (single) attestations we have `single_attestation` now.

Reply:
The description mentions `Attestation` container which implies aggregated attestations and block attestations and excludes single attestations. So it is kind of unambiguous in my opinion.